### PR TITLE
add Joe as a maintainer

### PR DESCRIPTION
**Pull request by lheagy**
add @jcapriot as a maintainer

### CODE REVIEW

**Comment by jcapriot** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ source:
 build:
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```
Comment (lheagy) [inline recipe/meta.yaml:14]: cc @jcapriot. I believe this is how we skip python 2.7 builds (see conda-forge/conda-smithy#1110)

**Comment by lheagy** [comment]
@conda-forge-admin please rerender

**Comment by lheagy** [comment]
@conda-forge-admin please rerender

**Pull request merged**